entity-framework/core/extensions/index.md

```diff
@@ -152,6 +152,12 @@ A high-performance second-level query cache. For EF Core: 2.
 
 [GitHub repository](https://github.com/SteffenMangold/EntityFrameworkCore.Cacheable)
 
+### EntityFrameworkCore.NCache
+
+NCache Entity Framework Core Provider is a distributed second level cache provider for caching query results. The distributed architecture of NCache makes it more scalable and highly available. 
+
```

Comment:
Please indicated supported EF Core versions (like all other extensions)

Reply:
"For EF Core: 2, 3." or similar

Reply:
I have added the supported version. NCache currently supports EF Core 2 and will soon be providing support with EF Core 3 but I will update that support later here.